float32 lat
float32 lng
float32 alt

uint8 quality
uint32 time_Y
uint32 time_M
uint32 time_D
uint32 time_h
uint32 time_m
uint32 time_s

